packages/manager/modules/pci/src/projects/new/components/pci-new-project-promotion-banner/index.scss

```diff
@@ -0,0 +1,20 @@
+.pci-new-project-promotion-banner {
+  color: white;
+
+  .promotion-banner {
+    cursor: pointer;
+    background: linear-gradient(180deg, #000e9c, #0050d7) 0% 0% no-repeat
+      padding-box;
+    width: 340px;
+
+    h4,
+    h5 {
+      color: white;
+    }
+  }
+  .pci-project-new-config-banner {
+    height: 125px;
+    background: linear-gradient(254deg, #284cc0, #0586ff);
+    padding: 15px 40px;
+  }
+}
```

Comment:
**nitpick:** Avoid custom css and use the utilities available in ui-kit.
You can use 

- `oui-background-p-600` or `oui-background-p-700` for background (Ref: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-helpers-backgrounds--primary)
- `oui-color-p-000` or `oui-color-p-050` for color (Ref: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-helpers-colors--primary)

Reply:
I agree that it would be better to use the UI-Kit components for the banner. 
However, the specification of this use case require the banner background to have this gradient and the text color to be white. 
So, I had to use a custom CSS class for this purpose.